tracker/src/plausible.js

```diff
@@ -364,6 +364,20 @@
     })
   }
 
+if (COMPILE_CONFIG && config.trackFormSubmissions) {
+  function trackFormSubmission(e) {
+    if (e.target.hasAttribute('novalidate') || e.target.checkValidity()) {
+      plausible('WP Form Completions', { props: { path: location.pathname } });
+    }
+  }
+
+  function attachFormSubmitTracking() {
+    document.addEventListener('submit', trackFormSubmission, true);
+  }
+
+  attachFormSubmitTracking();
```

Comment:
```suggestion
  document.addEventListener('submit', trackFormSubmission, true);
```

Unneeded indirection.